.github/workflows/tokamak-public-nightly.yml

```diff
@@ -635,31 +1216,304 @@ jobs:
+
+      - name: Stop EC2 runner 2
+        uses: machulav/ec2-github-runner@v2
+        with:
+          mode: stop
+          github-token: ${{ secrets.BOT_ACCESS_TOKEN }}
+          label: ${{ needs.start-runners.outputs.label-2 }}
+          ec2-instance-id: ${{ needs.start-runners.outputs.ec2-instance-id-2 }}
+
+      - name: Stop EC2 runner 3
+        uses: machulav/ec2-github-runner@v2
+        with:
+          mode: stop
+          github-token: ${{ secrets.BOT_ACCESS_TOKEN }}
+          label: ${{ needs.start-runners.outputs.label-3 }}
+          ec2-instance-id: ${{ needs.start-runners.outputs.ec2-instance-id-3 }}
+
+      - name: Stop EC2 runner 4
+        uses: machulav/ec2-github-runner@v2
+        with:
+          mode: stop
+          github-token: ${{ secrets.BOT_ACCESS_TOKEN }}
+          label: ${{ needs.start-runners.outputs.label-4 }}
+          ec2-instance-id: ${{ needs.start-runners.outputs.ec2-instance-id-4 }}
+
+      - name: Stop EC2 runner 5
+        uses: machulav/ec2-github-runner@v2
+        with:
+          mode: stop
+          github-token: ${{ secrets.BOT_ACCESS_TOKEN }}
+          label: ${{ needs.start-runners.outputs.label-5 }}
+          ec2-instance-id: ${{ needs.start-runners.outputs.ec2-instance-id-5 }}
+
+      - name: Stop EC2 runner 6
+        uses: machulav/ec2-github-runner@v2
+        with:
+          mode: stop
+          github-token: ${{ secrets.BOT_ACCESS_TOKEN }}
+          label: ${{ needs.start-runners.outputs.label-6 }}
+          ec2-instance-id: ${{ needs.start-runners.outputs.ec2-instance-id-6 }}
+
+      - name: Stop EC2 runner 7
+        uses: machulav/ec2-github-runner@v2
+        with:
+          mode: stop
+          github-token: ${{ secrets.BOT_ACCESS_TOKEN }}
+          label: ${{ needs.start-runners.outputs.label-7 }}
+          ec2-instance-id: ${{ needs.start-runners.outputs.ec2-instance-id-7 }}
+
+      - name: Stop EC2 runner 8
+        uses: machulav/ec2-github-runner@v2
+        with:
+          mode: stop
+          github-token: ${{ secrets.BOT_ACCESS_TOKEN }}
+          label: ${{ needs.start-runners.outputs.label-8 }}
+          ec2-instance-id: ${{ needs.start-runners.outputs.ec2-instance-id-8 }}
+
+      - name: Stop EC2 runner 9
+        uses: machulav/ec2-github-runner@v2
+        with:
+          mode: stop
+          github-token: ${{ secrets.BOT_ACCESS_TOKEN }}
+          label: ${{ needs.start-runners.outputs.label-9 }}
+          ec2-instance-id: ${{ needs.start-runners.outputs.ec2-instance-id-9 }}
+
+      - name: Stop EC2 runner 10
+        uses: machulav/ec2-github-runner@v2
+        with:
+          mode: stop
+          github-token: ${{ secrets.BOT_ACCESS_TOKEN }}
+          label: ${{ needs.start-runners.outputs.label-10 }}
+          ec2-instance-id: ${{ needs.start-runners.outputs.ec2-instance-id-10 }}
+
+  update-kustomize:
+    name: Update image version in kustomize manifest file
+    needs:
+      - multi-arch-image
```

Comment:
@nguyenzung Maybe I think we have to add some needs! we missed it,,,😢😍

Reply:
https://github.com/tokamak-network/tokamak-titan-infra/commit/8eae27e144e8405afe8951b01fca7336618d364c

Reply:
Thank you, Steven!